File: ekada-dotnet/models/MatchData.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ekada_dotnet.Models;

[Table("match_data")]
public class MatchData
{
    [Key]
    public int Id {get; set;}
    public string? MatchId { get; set; }
    public string? GameId { get; set; }
    public string? Team { get; set; }
    public string? Side { get; set; }
    public string? Name { get; set; }
    public string? Role { get; set; }
    public string? PlayerWin { get; set; }
    public string? Champion { get; set; }
    public string? Items { get; set; }
    public string? Trinket { get; set; }
    public int Kills { get; set; }
    public int Deaths { get; set; }
    public int Assists { get; set; }
    public int Gold { get; set; }
    public int VisionScore { get; set; }
    public string? SummonerSpells { get; set; }
    public string? Runes { get; set; }
    public int DamageToChampions { get; set; }
}

File: backend-express/models/MatchData.js
import { sequelize } from "../util/database.js";
import { DataTypes } from "sequelize";

const MatchData = sequelize.define(
    'MatchData',
    {
        MatchId: {
            type: DataTypes.STRING,
            allowNull: false,
        },
        GameId: {
            type: DataTypes.STRING,
        },
        Team: {
            type: DataTypes.STRING,
            allowNull: false,
        },
        Side: {
            type: DataTypes.STRING,
            allowNull: false,
        },
        Name: {
            type: DataTypes.STRING,
            allowNull: false,
        },
        Role: {
            type: DataTypes.STRING,
            allowNull: false,
        },
        PlayerWin: {
            type: DataTypes.STRING,
            allowNull: false,
        },
        Champion: {
            type: DataTypes.STRING,
            allowNull: false,
        },
        Role: {
            type: DataTypes.STRING,
            allowNull: false,
        },
        Items: {
            type: DataTypes.STRING,
            allowNull: false,
        },
        Trinket: {
            type: DataTypes.STRING,
            allowNull: false,
        },
        Kills: {
            type: DataTypes.INTEGER,
            allowNull: false,
        },
        Deaths: {
            type: DataTypes.INTEGER,
            allowNull: false,
        },
        Assists: {
            type: DataTypes.INTEGER,
            allowNull: false,
        },
        Gold: {
            type: DataTypes.INTEGER,
            allowNull: false,
        },
        VisionScore: {
            type: DataTypes.INTEGER,
            allowNull: false,
        },
        SummonerSpells: {
            type: DataTypes.STRING,
            allowNull: false,
        },
        Runes: {
            type: DataTypes.STRING,
            allowNull: false,
        },
        DamageToChampions: {
            type: DataTypes.INTEGER,
            allowNull: false,
        }
    }
)

export default MatchData;